typescript/sdk/src/providers/transactions/TransactionTypes.ts

```diff
@@ -0,0 +1,7 @@
+import {
+  EthersV5Transaction,
+  EthersV5TransactionReceipt,
+} from '../ProviderType.js';
+
+export type EV5Tx = EthersV5Transaction['transaction'];
+export type EV5Receipt = EthersV5TransactionReceipt['receipt'];
```

Comment:
these are already exported as types by the ethers library so not sure why we want aliases?

Reply:
Chatted offline– this was originally just a shorthand (sugar), but fine using the default EV5 types instead